src/common/java/net/minecraftforge/gradle/common/util/MinecraftRepo.java

```diff
@@ -168,14 +168,15 @@ protected File findPom(String side, String version, File json) throws IOExceptio
                 VersionJson meta = Utils.loadJson(json, VersionJson.class);
                 for (VersionJson.Library lib : meta.libraries) {
                     //TODO: Filter?
-                    builder.dependencies().add(lib.name, "compile");
                     if (lib.downloads.classifiers != null) {
                         if (lib.downloads.classifiers.containsKey("test")) {
                             builder.dependencies().add(lib.name, "test").withClassifier("test");
                         }
-                        if (lib.natives != null && lib.natives.containsKey(CURRENT_OS)) {
+                        if (lib.natives != null && lib.natives.containsKey(CURRENT_OS) && !lib.getArtifact().getName().contains("java-objc-bridge")) {
                             builder.dependencies().add(lib.name, "runtime").withClassifier(lib.natives.get(CURRENT_OS));
```

Comment:
Is there a better way then hard codding the specific file name? Seems REALLY prone to breaking.
Also, any idea what the official natives method for gradle is? I dont think us manually extracting things is the best approach.

Reply:
I agree that this is not a great solution. With that said, I do not know what other options we have here. Once this dependency is added to the pom, gradle will try to resolve it and fail. Even with Mojang's repo added it will fail because their repo does not use proper classifiers.

Also, I did search around quite a bit. It does not seen like gradle has any builtin support for natives, so I am not sure if we can get around manually extracting them.